browser-test/src/application_navigation.test.ts

```diff
@@ -154,23 +154,18 @@ describe('Applicant navigation flow', () => {
 
       // Verify we are on program list page.
       expect(await page.innerText('h1')).toContain('Get benefits')
-      expect(
-        await page.locator('a:has-text("External site")').getAttribute('href'),
-      ).toEqual('https://external.com')
-      await validateAccessibility(page)
-      await validateScreenshot(page, 'program-list-page')
-    })
-
-    it('verify program details page', async () => {
-      const {page, applicantQuestions} = ctx
-      await loginAsGuest(page)
-      await selectApplicantLanguage(page, 'English')
-      await applicantQuestions.clickProgramDetails(programName)
 
-      // Verify we are on program details page. Url should end in "/programs/{program ID}"
-      expect(page.url()).toMatch(/\/programs\/[0-9]+$/)
+      const cardHtml = await page.innerHTML(
```

Comment:
**suggestion:** Here we verify that the program with the external link has the correct URL. Could we also do a validation that the first program (with program details link) has the correct URL? Maybe by leaving in the clickProgramDetails() and asserting that way?

Reply:
We cannot directly test it as it opens in a new tab. I have added the verify_program_details test and managed to test the URL using the popup functionality of TS.

Reply:
Sounds good, thank you for adding that!!